src/Ledger/PParams.lagda

```diff
@@ -1,4 +1,9 @@
-\section{Protocol parameters}
+\section{Protocol parameters\protect\footnotemark}
+\label{sec:protocol-parameters}
+\footnotetext{See also: \href{https://github.com/cardano-foundation/CIPs/tree/master/CIP-1694}{github.com/cardano-foundation/CIPs/CIP-1694}}
```

Comment:
Instead of referencing CIP-1694 here, we should add a small section to the introduction that states that most things in this document follow what's outlined there. Otherwise we'll end up referencing it tens of times.

Reply:
I've added a brief paragraph to the start of the introduction.  Feel free to revise or suggest revisions.